src/Controls/tests/DeviceTests/Elements/Toolbar/ToolbarTests.cs

```diff
@@ -117,6 +117,43 @@ public async Task ToolbarTitle()
 			});
 		}
 
+		[Fact(DisplayName = "Toolbar Uses TabbedPage Title When Set")]
+		public async Task ToolbarTabbedPageTitle()
```

Comment:
I think it's alright to remove these. We have tests that validate that `Toolbar.Title` makes it to the platform. So, I think the unit tests plus those tests cover what we need.

Reply:
Ok, removed and added more tests validating the title changing tabs etc.